### PR TITLE
Add PW-Turn evaluation metrics and model comparisons to documentation

### PR DESCRIPTION
**Pull request by Rebecca-Qian**
**Patch description**
Add documentation for using PW-Turn for different use cases and reproducing results in the human eval comparison paper:
- How to run different model comparisons eg. BlenderBot 3B vs. Reddit 3B
- How to change the evaluation metric, eg. selecting conversational responses that are more engaging

**Testing steps**
N/A

### CODE REVIEW

File: parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md:49]: nit: this second model is "BlenderBot3B-M0" in the paper

File: parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md:69]: Hmm it's probably worth saying that we provide in `task_config/` an example onboarding JSON for all 3 eval metrics above

Nit: `onboard_task_data__engaging.json` --> `task_config/onboard_task_data__engaging.json`?

File: parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md:71]: 👍

File: parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md:64]: The user also needs to set `mephisto.blueprint.task_question` in addition to `annotation_question`: the former is used during the conversation itself and the latter is used only during onboarding

**Review by EricMichaelSmith — COMMENTED** [review]
Great to have! Just a bunch of nitpicky comments

File: parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md
Comment (Rebecca-Qian) [reply parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md:64]: Good point

**Review by EricMichaelSmith — APPROVED** [review]
Great, thanks for making these revisions! Approving now, but take a look at the failing pre-commit CI check - that should be fixed before merging in

**Pull request merged**
